web/pdf_viewer.js

```diff
@@ -144,6 +146,7 @@ var PDFViewer = (function pdfViewer() {
     this.renderInteractiveForms = options.renderInteractiveForms || false;
     this.enablePrintAutoRotate = options.enablePrintAutoRotate || false;
     this.renderer = options.renderer || RendererType.CANVAS;
+    this.apiIgnoreErrors = options.apiIgnoreErrors || false;
```

Comment:
It's `true` at https://github.com/mozilla/pdf.js/pull/8176/commits/8bbca5999ea2544c032dc2ab25d93e949fa2617a#diff-a0d82fd09027b92c6e7bed99dc735626 and at https://github.com/mozilla/pdf.js/pull/8176/commits/8bbca5999ea2544c032dc2ab25d93e949fa2617a#diff-2930ccbe813e302a92f28334307aa4f6, but `false` in other places like here, which is confusing. I personally think we can make `true` the default to recover as much as possible, like other readers and PDF.js in other places in the code.